src/records/RecCore.cc

```diff
@@ -576,8 +576,7 @@ RecLookupMatchingRecords(unsigned rec_type, const char *match, void (*callback)(
         callback(&tmp, data);
       }
     }
-    // all done for metrics
-    return REC_ERR_OKAY;
+    // Fall through to return any matching string metrics
```

Comment:
so, the `g_records` is still holding metrics??

Reply:
It holds string metrics, yes

Reply:
For example:
```
traffic_ctl metric get proxy.process.version.server.long
```

works but:
```
traffic_ctl metric match proxy.process.version.server.*
```

does not